app.js

```diff
@@ -1,5 +1,44 @@
 // import functions and grab DOM elements
+import { countsAsAYes } from './utils.js';
 
+const resultDisplay = document.getElementById('results-display');
+const quizButton = document.getElementById('quiz-button');
 // initialize state
 
-// set event listeners to update state and DOM
\ No newline at end of file
+// set event listeners to update state and DOM
+quizButton.addEventListener('click', () => {
+
+    alert('Hello, and welcome to my quiz!!');
+
+    const confirmation = confirm('Do you wish to continue?');
+    if (!confirmation) return;
+
```

Comment:
This all looks good but you'll only ever need one line of whitespace, never two!